### DISCUSSION TITLE
Can lifelines call a frailty model?

### DISCUSSION THREAD

**Discussion by pengfei2000** [CamDavidsonPilon/lifelines#1530] [Q&A] [answered]
Can lifelines call the frailty model? If so, I would appreciate any help with the exact method or code. If not, is there an alternative method.

**Comment by CamDavidsonPilon** [comment, marked as answer]
The Cox frailty model? No, it's not implemented. It's a lot of work to implement, and I've just not had time to do so.

**Reply by pengfei2000** [reply]
Thank you for your answer. Then is there a similar model in the library that can take into account heterogeneity?

**Reply by spyrosiatrop** [reply]
> The Cox frailty model? No, it's not implemented. It's a lot of work to implement, and I've just not had time to do so.

Hi, I have seen recently someone using the cluster argument to pass the subject_id as a way to introduce the frailty concept (https://fjs.fudutsinma.edu.ng/index.php/fjs/article/view/1970/1550). Is this working according to you? If so, can we pass more than one clustering levels? Thanks!

**Reply by jamescporter** [reply]
Would it be hard to implement frailty via the cluster argument?
